### PR TITLE
#162948871 Users' registration and verification

### PR DESCRIPTION
**Pull request by danuluma**
#### What does this PR do?
Implements users' registration

#### Description of Task(s) to be completed?
* Adds user registration feature to the frontend
* Users receive a verification link in their emails

#### How should this be manually tested?
* Pull the branch `git fetch origin ft-registration-162948871:ft-registration-162948871`
* Checkout to the branch `git checkout ft-registration-162948871`
* Install dependencies `npm install`
* Run the server `npm start`

* Alternatively, simply click on this [link](https://ah-alpha-frontend-staging-pr-9.herokuapp.com/) to view the deployed version

### Screenshots
image

#### What are the relevant pivotal tracker stories?
[#162948871](https://www.pivotaltracker.com/story/show/162948871)

### CODE REVIEW

**Review by Deekerubo — APPROVED** [review]

**Review by jamesbeamie — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
+import { ToastContainer } from "react-toastify";
+import VerifyEmail from "./components/authentication/EmailVerification";
+import "react-toastify/dist/ReactToastify.css";
 
 // Class component
-import Home from "./components/Home";
 import store from "./redux/store";
 import "./App.scss";
-import "semantic-ui-css/semantic.min.css";
+// import "semantic-ui-css/semantic.min.css";
```
Comment (Betty-Kebenei) [inline src/App.js]: Kindly remove this line

File: src/App.scss [hunk 2]
```diff
@@ -1,25 +1,104 @@
...
+  padding: 25px;
+}
+
+a:hover {
+  cursor: pointer;
+}
+#login-button {
+  width: 100%;
+  background: transparent;
+  /* background-color: col !important; */
```
Comment (Betty-Kebenei) [inline src/App.scss]: Kindly remove all commented code

File: src/redux/reducers/index.js [hunk 3]
```diff
@@ -1,7 +1,11 @@
 import { combineReducers } from "redux";
 import loginReducer from "./loginReducer";
+// import { loginReducer } from "./loginReducer";
```
Comment (Betty-Kebenei) [inline src/redux/reducers/index.js]: This too

File: src/redux/reducers/loginReducer.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
 /* eslint-disable no-case-declarations */
 /* eslint-disable prefer-destructuring */
 import { LOGIN_REQUEST, LOGIN_ERROR, LOGIN_SUCCESS } from "../../actions/types";
-import initialState from "../store/initialState";
+// import initialState from "../store/initialState";
```
Comment (Betty-Kebenei) [inline src/redux/reducers/loginReducer.js]: Another one

**Review by Betty-Kebenei — APPROVED** [review]
Good work

**Pull request merged**